core/jazz_acl/components/utils/vault.js

```diff
@@ -185,8 +185,8 @@ function addOrRemoveAllAdminUsersToSafe(environmentDetails, configData, serviceI
 
     for (let key in environments) {
       const environment = environments[key];
-      if (environment.metadata && environment.metadata.SAFE_DETAILS)  //TODO to check the case after the environment changes added
-        processEachPromises.push(processEachPromise(environment.metadata.SAFE_DETAILS.name, configData, serviceId, authToken, adminUsers));
+      if (environment.metadata && environment.metadata.SAFE)  //TODO to check the case after the environment changes added
+        processEachPromises.push(processEachPromise(environment.metadata.SAFE.name, configData, serviceId, authToken, adminUsers));
```

Comment:
should this be `environment.metadata.safe`?

Reply:
same as above explanation